python/tests/test_sklearn_vw.py

```diff
@@ -232,6 +232,24 @@ def test_delete(self):
         raw_model = VW()
         del raw_model
 
+    def test_save_load(self, tmp_path):
+        train_file = os.path.realpath(os.path.join(tmp_path, "train.model"))
+
+        X = [[1, 2], [3, 4], [5, 6], [7, 8]]
+        y = [1, 2, 3, 4]
+
+        model = VWRegressor(l=100)
+        model.fit(X, y)
+        before_saving = model.predict(X)
+
+        model.save(train_file)
+
+        model2 = VWRegressor(l=100)
+        model2.load(train_file)
+        after_loading = model2.predict(X)
+
+        assert all([a == b for a, b in zip(before_saving, after_loading)])
```

Comment:
Simplify to?
   `assert before_loading == after_loading`

Reply:
I thought the same thing but when I do it fails with:
`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`